web_src/js/features/repo-issue.js

```diff
@@ -287,6 +287,20 @@ export function initRepoPullRequestMergeInstruction() {
   });
 }
 
+export function initRepoPullRequestAllowEditsFromMaintainers() {
```

Comment:
Minor note: It seems like everything is done via `<form>` etc... instead of using javascript to do these request. Not sure if this should differ.

CC @silverwind

Reply:
I see no form above, what do you mean?

Reply:
Ah, new PR is using a form. I guess it's okay to send it in the form there as there is no ajax API available during issue creation, I assume.

Reply:
Oh I meant, that all the other options/fields are implemented via `<form><input ...>`

Reply:
I think we probably need to support both variants. Using `<form>` for new PRs and using JS to update existing PR.